src/modules/task/internal/task-util.ts

```diff
@@ -0,0 +1,13 @@
+import { Task } from '../types';
+import { TaskDB } from './store/task-db';
+
+export default class TaskUtil {
+  public static convertTaskDBToTask(taskDb: TaskDB): Task {
+    const task = new Task();
+    task.id = taskDb._id.toString();
+    task.account = taskDb.account.toString();
+    task.active = taskDb.active;
```

Comment:
Remove

Reply:
@jjalan  sir, what to remove here exactly ?, this piece of code is prewritten to convert task from DB into plain values.